README.md

```diff
@@ -34,3 +34,7 @@ Run ```make emulator/start``` to start the emulator and ```make emulator/version
 [GRUB Theme Tutorial](http://wiki.rosalab.ru/en/index.php/Grub2_theme_tutorial)
 
 [Gnome-Look GRUB Themes](https://www.gnome-look.org/browse/cat/109/order/latest/)
+
+# Fork
+
+Forked by slashpotato to add more icons
```

Comment:
Could you remove this section? It doesn't add extra value to the `README.md` imo, since your contribution will be visible in the commit history :)